playpen/ansible/dev-playbook.yml

```diff
@@ -6,4 +6,5 @@
     - core
     - db
     - qpidd
-    - dev
\ No newline at end of file
+    - dev
+    - lazy
```

Comment:
I just noticed this a few minutes ago, too. I'll add it in both until one or the other passes into the ether.